src/Trixi.jl

```diff
@@ -15,6 +15,12 @@ See also: [trixi-framework/Trixi.jl](https://github.com/trixi-framework/Trixi.jl
 """
 module Trixi
 
+using Preferences: @load_preference, set_preferences!
+const _PREFERENCE_SQRT = @load_preference("sqrt", "sqrt_Trixi_NaN")
+const _PREFERENCE_LOG = @load_preference("log", "log_Trixi_NaN")
+const _PREFERENCE_POLYESTER = @load_preference("polyester", true)
+const _PREFERENCE_LV = @load_preference("loop_vectorization", true)
```

Comment:
```suggestion
const _PREFERENCE_LOOPVECTORIZATION = @load_preference("loop_vectorization", true)
```
If you don't mind, I'd prefer to make this more self-descriptive for unfamiliar users